src/main/java/github/scarsz/discordsrv/listeners/DiscordConsoleListener.java

```diff
@@ -100,9 +102,16 @@ public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
                 if (DiscordSRV.config().getBoolean("CancelConsoleCommandIfLoggingFailed")) return;
             }
         }
+        // Create DiscordConsoleCommandPreProcessEvent, and stop the command from being run if the API user wants to stop the command from being run
+        DiscordConsoleCommandPreProcessEvent consoleEvent = DiscordSRV.api.callEvent(new DiscordConsoleCommandPreProcessEvent(event, event.getMessage().getContentRaw(), true));
+        if(consoleEvent.isCancelled()) return;
```

Comment:
More spaces!

Reply:
Still needs spaces between the `if` and the `{`

Reply:
![image](https://user-images.githubusercontent.com/58291118/124382480-5b6b5080-dcd0-11eb-8e99-a11c4c5d68e0.png)
Here is how it currently looks

Is it okay or should I change something?

Reply:
If you highlight what you've changed in intellij and do `ctrl+alt+l` it'll format it automatically

Reply:
ah ok lol. Didn't realise you were talking about the space between the if and the start of the parenthesis

Reply:
fixed

Reply:
also added a space in the DiscordChatListener at the same spot